### ISSUE TITLE
Add Remove For User Actions to Frontend

### ISSUE THREAD

**Issue by jvc56** [domino14/liwords#689]

**Comment by domino14** [comment]
what's an example?

**Comment by jvc56** [comment]
We should be able to remove user actions without applying another action to replace it that expires in 2 seconds.

**Comment by andy-k** [comment]
that's by design, the transient action is an opportunity to write a meaningful comment

**Comment by jvc56** [comment]
Good point, though it would be nice if we could write comments without applying an entire action.
